langserver/server.go

```diff
@@ -164,6 +169,30 @@ func (s *server) connectPrometheus(url string) error {
 			return err
 		}
 
+		bodyBytes, _ := ioutil.ReadAll(resp.Body)
+		jsonResponse := map[string]interface{}{}
+
+		err = json.Unmarshal(bodyBytes, &jsonResponse)
+
+		if err == nil {
+			respData := jsonResponse["data"].(map[string]interface{})
```

Comment:
Unchecked typecasts can panic.

We shouldn't do this with data we get from a remote host.